api/src/main/java/jakarta/data/Sort.java

```diff
@@ -152,67 +158,67 @@ public boolean isDescending() {
     /**
      * Create a {@link Sort} instance
      *
-     * @param <T>        entity class of the sortable property.
-     * @param property  the property name to order by
-     * @param direction the direction in which to order.
-     * @param ignoreCase whether to request a case insensitive ordering.
+     * @param <T>           entity class of the sortable entity attribute.
+     * @param attributeName name of the entity attribute to order by
+     * @param direction     the direction in which to order.
+     * @param ignoreCase     whether to request a case insensitive ordering.
      * @return a {@link Sort} instance. Never {@code null}.
      * @throws NullPointerException when there is a null parameter
      */
-    public static <T> Sort<T> of(String property, Direction direction, boolean ignoreCase) {
+    public static <T> Sort<T> of(String attributeName, Direction direction, boolean ignoreCase) {
         Objects.requireNonNull(direction, "direction is required");
-        return new Sort<>(property, Direction.ASC.equals(direction), ignoreCase);
+        return new Sort<>(attributeName, Direction.ASC.equals(direction), ignoreCase);
     }
 
     /**
      * Create a {@link Sort} instance with {@link Direction#ASC ascending direction}
      * that does not request case insensitive ordering.
      *
-     * @param <T>      entity class of the sortable property.
-     * @param property the property name to order by
+     * @param <T>           entity class of the sortable entity attribute.
+     * @param attributeName name of the entity attribute to order by
```

Comment:
`attributeName`sounds odd, why not just `attribute`?

Reply:
Thanks for the feedback. I can switch it to that. Here is the commit for it, https://github.com/jakartaee/data/pull/935/commits/d82c0449ce9d12950aa13046847fec01943b5744